### ISSUE TITLE
'apt-daily.service' block other apt processes during startup

### ISSUE THREAD

**Issue by jkrue** [BiBiServ/bibigrid#61]
Since BiBiGrid now supports standard Ubuntu cloud images, we must stop/disable apt-daily.service The apt-daily.service is started by cloud-init during initialisation. The services itself is triggered immediately and  prevents other apt processes to run until they are finished. In our case  a running apt-daily.service prevents BiBiGrid to install ansible and run the BiBigrid specific playbooks. 

A possible solution is discussed [here](https://unix.stackexchange.com/questions/315502/how-to-disable-apt-daily-service-on-ubuntu-cloud-vm-image/315517).  It seem that only a sledge-hammer helps :
```
systemctl stop apt-daily.service
systemctl disable apt-daily.service
systemctl kill --kill-who=all apt-daily.service

# wait until `apt-get updated` has been killed
while ! (systemctl list-units --all apt-daily.service | fgrep -q dead)
do
  sleep 1;
done
```

**Comment by jkrue** [comment]
It seems that we must also stop/disable apt-daily.timer.
```
systemctl stop apt-daily.timer
systemctl disable apt-daily.timer
```

**Comment by jkrue** [comment]
Solved with commit 7fe1fba
